core/ui/PageTemplate.tid

```diff
@@ -1,7 +1,8 @@
 title: $:/core/ui/PageTemplate
+tags: $:/tags/Layout
```

Comment:
Need to keep this, so it shows in launchers.

Reply:
Hi @linonetwo rethinking the tag on `$:/tags/ui/PageTemplate` is really a separate topic, and should go into its own PR with some documentation updates.

Reply:
OK, let's discuss this later. If add this tag, may need additional `list` on `$:/tags/Layout` tiddler to keep `$:/core/ui/PageTemplate` on top.

Reply:
Thanks @linonetwo